examples/python/alphabet_partitioned/README.md

```diff
@@ -34,7 +34,9 @@ The `Decoder`'s `decode(...)` method creates a `Votes` object with the letter be
 
 ## Running Alphabet Partitioned
 
-In order to run the application you will need Machida, Giles Sender, Giles Receiver, and the Cluster Shutdown tool. To build them, please see the [Linux](/book/getting-started/linux-setup.md) or [Mac OS](/book/getting-started/macos-setup.md) setup instructions.
+In order to run the application you will need Machida, Giles Sender, Giles Receiver, and the Cluster Shutdown tool. To build them, please see the [Linux](/book/getting-started/linux-setup.md) or [Mac OS](/book/getting-started/macos-setup.md) setup instructions. Alternatively, they could be run in Docker, please see the [Docker](/book/getting-started/docker-setup.md) setup instructions.
```

Comment:
Why are we telling people to go look at the docker installation instructions here and in other similar places? i would have expected to send them to the page explaining how to run a wallaroo app in docker.....

Reply:
@dipinhora this has been addressed, let me know if it makes more sense now